### ISSUE TITLE
gulp-riot broken now?

### ISSUE THREAD

**Issue by phillip-haydon** [e-jigsaw/gulp-riot#65]
I was running (Current) and just updated everything to Wanted/Latest.

```
Package              Current  Wanted  Latest
babel                 6.1.18  6.3.13  6.3.13
babel-core             6.2.1  6.3.17  6.3.17
babel-preset-es2015   6.1.18  6.3.13  6.3.13
gulp-riot              0.4.1   0.4.2   0.4.2
```

With the task as:

```
gulp.task('run-riot', function() {
    return gulp.src([
        './src/Riot-Tags/**/*.tag'
    ])
    .pipe(riot({
        compact: true,
        whitespace: false,
        type: 'babel'
    }))
    .pipe(concat('riot-tags.js'))
    .pipe(gulp.dest('./src/content/js'));
});
```

Everything was working great until I updated. Now it fails to compile all the es6 stuff to es5. 

Basically if I wrote:

```
this.like = (e) => {

};
```

It doesn't compile it.

Is this a bug or am I doing something wrong since updating?

**Comment by phillip-haydon** [comment]
OK, I managed to fix it. 

http://riotjs.com/guide/compiler/#babel-6

I followed the guide:
1. install our babel-preset-es2015-riot
   
   `npm install babel-preset-es2015-riot --save-dev`
2. install babel-core as well 
   
   `npm install babel-core --save-dev`
3. create a .babelrc file containing the preset id
   
   `{ "presets": ["es2015-riot"] }`

Then ran gulp again. Bam everything working perfectly.
